package.json

```diff
@@ -46,6 +46,7 @@
   },
   "devDependencies": {
     "@govflanders/vl-ui-core": "^3.9.2",
-    "vl-ui-util": "^5.2.4"
+    "vl-ui-util": "^5.2.4",
+    "yargs": "^15.3.1"
```

Comment:
Vormde het een probleem dat yargs transitief met vl-ui-util mee kwam? Handiger als we de externe dependencies maar op één plaats hebben staan.

Reply:
Meegenomen in de PR van util.